test/libsolidity/ASTJSON/address_payable.json

```diff
@@ -281,7 +281,7 @@
                       "name": "this",
                       "nodeType": "Identifier",
                       "overloadedDeclarations": [],
-                      "referencedDeclaration": 68,
+                      "referencedDeclaration": -28,
```

Comment:
I wonder if there is an easy way to pull out these changes into a new PR, but I fear it would be too much effort.